canonical-web-and-design/hiring.canonical.com
# Add warning text when no data exist

## Done
Warning text is displayed when no data exist.

## QA
- Run server: `dotrun`
- Open "Applications review" from `http://0.0.0.0:8501/stages`
- Select a hiring lead with no data (ie Andrey Grebennikov)
- Verify that instead of "An error occurred",  "No data" message is displayed

## Issue
Fixes #144
